### PR TITLE
Script consolidation

### LINKED ISSUE #13: Is DFT code deterministic?
I ran the DFT code twice from commit f22d93364ebda98ab476483fc95f7bcac6019696 in offline and received differing results from both runs. This is troubling considering the input should have been identical.

command used:
```
python LWA_bifrost_DFT.py --offline --tbnfile=/data5/LWA_SV_data/data_raw/TBN/Jupiter/058161_000086727 --imagesize 64 --imageres 1.79057 --nts 512 --channels 4 --accumulate 50 --ints_per_file 40 --out_dir dft/run_?
```

See attached jupyter notebook output for more information. 
[dft_comparison.pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5228656/dft_comparison.pdf)

### PR DESCRIPTION
**Pull request by mkolopanis**
Removes the redundant script lwa_bifrost_DFT.py  and adds the DFT functionality to the main script. 
A bit of formatting for readability.
Added the synchronization for the DFT accumulation. closes #13 

I removed the TransposeOp. It served no purpose anymore since the transpose appears to happen upstream? It was no longer in the original script and with the changes it was not needed for the DFT one.

See comparison with master in this notebook. The DFT shouldn't match but does now between multiple runs.

[script_consolidate.pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5234622/script_consolidate.pdf)
multiple runs of the dft code on the main branch
[dft_comparison.pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5234624/dft_comparison.pdf)
multiple runs of dft code with this branch.
[dft_comparison_update.pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5234625/dft_comparison_update.pdf)

### CODE REVIEW

**Comment by mkolopanis** [comment]
@KentJames Do you want to take a look to make sure I got the DFT code moved over correctly?

**Comment by mkolopanis** [comment]
Here's another notebook comparison. I ran the code in a python 3 environment (the 'py3_test' file). 
[script_consolidate (2).pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5336284/script_consolidate.2.pdf)

**Comment by KentJames** [comment]
Sorry I clearly missed this. I’ll take a look.

File: LWA/LWA_bifrost.py [hunk 1]
```diff
@@ -1150,6 +1249,343 @@ def main(self):
...
+
+                lm_matrix = numpy.zeros(shape=(self.skymodes1d, self.skymodes1d, 3))
+                lm_step = 2.0 / self.skymodes1d
+                for i in numpy.arange(lm_matrix.shape[0]):
+                    for j in numpy.arange(lm_matrix.shape[0]):
+                        lm_matrix[i, j] = numpy.asarray([i * lm_step - 1.0, j * lm_step - 1.0, 0.0])
+                lm_vector = lm_matrix.reshape((self.skymodes, 3))
+                print("shapes: locs {locs}, phases {phases}".format(
+                    locs=locs.shape, phases=phases.shape,
+                ))
```
Comment (adampbeardsley) [inline LWA/LWA_bifrost.py]: I think this print statement is new. Do we want it?

**Review by adampbeardsley — COMMENTED** [review]
I've looked it over and the only comment I have is that there is a new print statement. I think it's fine to leave it in, unless it was a diagnostic that we don't normally need.
I'll hold off on "approving" in case James wants to have a look.
Nice work Matt!

**Comment by mkolopanis** [comment]
I think it is new as well probably want to remove it. It most likely was a
put in to help figure out some int vs float issues I had.

**Comment by KentJames** [comment]
Everything looks kosher to me. And thanks for going through and commenting everything properly Matt... clearly that fell by the wayside a bit in my rush to write up. 

I've been thinking for a while about how we can use these both together.. EPIC for the dense core of antennas, and then EPIC-DFT for any outrigger antennas.. seems like we aren't far off now you've merged the two strands into one whole.

**Comment by mkolopanis** [comment]
I think I added comments as I figured out what things did :smile: it was educational for me.


Here's a notebook showing the vectorized version of the code I changed produces identical outputs.
[numpize_for_loops.pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5349435/numpize_for_loops.pdf)

And that the images are identical 
[dft_comparison (1).pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5349454/dft_comparison.1.pdf)

also this shows images are identical in python 2 and 3 now 
[script_consolidate (3).pdf](https://github.com/epic-astronomy/LWA_EPIC/files/5349469/script_consolidate.3.pdf)

To be fair in all the comparison I only opened a single file and looked at all the images it contained. Could be more thorough and compare across all frequencies if there is a desire.

**Comment by mkolopanis** [comment]
@adampbeardsley do you want to take another look over this after the DFT blessing was bestowed?

**Pull request merged**